### ISSUE TITLE
nicer looking palm trees

### ISSUE THREAD

**Issue by kfarr** [kfarr/streetmix3d#56]
for example
https://www.turbosquid.com/3d-models/3d-model-palm-tree/960474

**Comment by kfarr** [comment]
![image](https://user-images.githubusercontent.com/470477/87866414-7b2a2d00-c936-11ea-8447-4fefb7be19a2.png)
trying 2 plane trees from @ex3d
![image](https://user-images.githubusercontent.com/470477/87866420-9301b100-c936-11ea-93f6-e8008801ef1c.png)
but doesn't look great from top down
![image](https://user-images.githubusercontent.com/470477/87866422-a3b22700-c936-11ea-9e31-59acfa7403b1.png)

next try VR mode

**Comment by kfarr** [comment]
In VR it is not bad. It's obviously not a real looking tree, but neither was the low poly version. Might be worth exploring the middle poly options now.

**Comment by kfarr** [comment]
I can't get the other higher poly (non 2 plane) versions to work in blender
![image](https://user-images.githubusercontent.com/470477/87866629-f5f44780-c938-11ea-8793-54a881c77a88.png)

**Comment by EX3D** [comment]
Nice! Hey tell you what I´m behind in the task of upgrading my trees. I´ll add pals for good measure to. When they are ready I´ll share the with you.

**Comment by EX3D** [comment]
> 
> 
> I can't get the other higher poly (non 2 plane) versions to work in blender
> ![image](https://user-images.githubusercontent.com/470477/87866629-f5f44780-c938-11ea-8793-54a881c77a88.png)

I´ll check if you have png or just the diffusion maps. Hang in there.

**Comment by EX3D** [comment]
> but doesn't look great from top down
> ![image](https://user-images.githubusercontent.com/470477/87866422-a3b22700-c936-11ea-9e31-59acfa7403b1.png)
> 
> next try VR mode

 ![image](https://github.com/EX3D/aframe-InstancedMesh/blob/master/aframe%20instancedmesh%20Example.jpg)




Yep those trees are made for large quantities of trees in "forest like setups". I think you need a higher poly version for "urban" or "standalone" settings. I´ll check some options.

**Comment by EX3D** [comment]
![2020-07-19_002311](https://user-images.githubusercontent.com/66544580/87867771-1bdc1500-c956-11ea-97ad-06657edd3dfd.png)

To avoid this try to use an alphavalue of 0.05

**Comment by kfarr** [comment]
Thanks for all the feedback @EX3D! Upgraded tree models would definitely be welcome whenever you get to it. This is already going in the right direction, and the alphatest tip worked great!

**Comment by kfarr** [comment]
Stashed this in a branch for now: https://github.com/kfarr/streetmix3d/tree/new-trees

**Comment by kfarr** [comment]
Other potential trees:
- https://sketchfab.com/3d-models/tree-1-df9269d8076c4e48880254327f0a77c2 6.2k tris
- https://sketchfab.com/3d-models/hill-top-tree-25f7a45343724eab93520a966f8c7843 1.9k tris
- https://sketchfab.com/3d-models/trees-low-poly-1d2dcca2ccb1496c85b7cc5789a2a261 5.4k tris
- https://sketchfab.com/3d-models/low-poly-tree-606d11eec2504972aa5e455c88879872 7k tris (2.3k per)

**Comment by EX3D** [comment]
Yep I have not forgotten that. By february the 10th I should have time to
continue that project.

Cheers
[image: www.ex3d.com] <http://www.ex3d.com>
